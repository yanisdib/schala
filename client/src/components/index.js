// COMPONENTS EXPORTS

export { default as Header } from './Header/Header';

export { default as Button } from './Button/Button';
export { default as MenuButton } from './MenuButton/MenuButton';
export { default as DisplayIdentifiedButton } from './DisplayIdentifiedButton/DisplayIdentifiedButton';

export { default as Tag } from './Tag/Tag';
export { default as Pointer } from './Pointer/Pointer';
export { default as Bubble } from './Bubble/Bubble';

export { default as PictureIdentifier } from './PictureIdentifier/PictureIdentifier';