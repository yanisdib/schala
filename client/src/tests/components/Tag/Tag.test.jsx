import { shallow } from 'enzyme';

import { Tag } from '../../../components';


describe('Tag Component', () => {
    let wrapper;
    const { variant } = Tag.defaultProps;

    beforeEach(() => wrapper = shallow(<Tag>Hello</Tag>))

    it('renders properly', () => expect(wrapper).toMatchSnapshot())

    it('renders with children text properly', () => expect(wrapper.text()).toEqual('Hello'))

    it('has a defaultProps', () => expect(variant).toEqual('primary'))
})