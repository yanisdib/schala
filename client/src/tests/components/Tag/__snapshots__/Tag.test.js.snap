// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag Component renders properly 1`] = `
<div
  className="sc-jSgvzq fGdppx"
>
  Hello
</div>
`;
